server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseCreateDTO.java

```diff
@@ -0,0 +1,72 @@
+package com.objectcomputing.checkins.services.pulseresponse;
+
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+import java.time.LocalDate;
+
+@Introspected
+public class PulseResponseCreateDTO {
+
+    @NotNull
+    @Schema(required = true, description = "id of the associated submissionDate")
+    private LocalDate submissionDate;
+
```

Comment:
The id is a date?

Reply:
It is date, updated the description. thanks.